CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# CatPhotoApp Contributing
+
+### Fork the repo
+
+### Clone your fork
+
+###  Branch off master
+
+#### `npm install`
+
+#### `gulp`
+
+This will get you running on `localhost:8080`
+
+[Squash](https://www.freecodecamp.com/wiki/en/git-squash/) your commits into one.
```

Comment:
You don't really need to add this, most contributors for this project will know git.